### ISSUE TITLE
Could not reach Twitch servers, check your internet connection or try again

### ISSUE THREAD

**Issue by Mte90** [jkbz64/AYAYA#1]
I am getting this error after compiling.

ANyway required to install `libmpv-dev` on linux, also doesn't detect that I have already youtube-dl.

**Comment by jkbz64** [comment]
As is clearly visible in the commits history the project was not updated for a quite a long time and it was written back when Twitch api did not require Bearer token for "public" endpoints such as top twitch games, streams, etc so it just gets 401 response and there is currently nothing that can be done about it - the client is pretty much dead unless someone bothers implementing log in mechanism and whatnot.

Indeed libmpv-dev or other distro equivalent development package is required to compile as is stated in linux dependencies section. 

There are no global youtube-dl availability checks - just local copy of binary that gets downloaded from the official website once, it is kinda weird looking back then when it asks if you want to download it when it doesn't even use global binary, welp.

Thanks for reporting the issue though

**Comment by Mte90** [comment]
I saw that there was a commit in october 2020 and the Twitchqt component got updates too very recently so I was thinking that it was updated.

**Comment by Mte90** [comment]
Looking at twitchqt the support is already there https://github.com/jkbz64/TwitchQt/commit/a73a7bb83aec2398f4a3ce6737ba83ba9d75f1db but is missing in this client

**Comment by jkbz64** [comment]
Yes, someone was using api wrapper in his own project(?) that this project depends on too and decided to add support for bearer token and send a pr in my repo but the client was and is unmaintained for like 2 years.

I don't really have much time to develop/maintain it and at this point there are probably better alternatives so it is dead project (thought the api was too but someone did actually use it)

**Comment by Mte90** [comment]
We forked it and we will try to add that support and do some other changes https://github.com/Mte90/twitchy
